config/content.d/developer.rackspace.com.json

```diff
@@ -33,7 +33,8 @@
           "/docs/private-cloud/rpc/v13/": "https://github.com/rcbops/privatecloud-docs/v13/",
           "/docs/private-cloud/rpc/v12/": "https://github.com/rcbops/privatecloud-docs/v12/",
           "/docs/private-cloud/release-notes/": "https://github.com/rcbops/rpc-openstack/master/",
-          "/docs/rkaas/1.0.x/": "https://github.com/rcbops/mk8s/1.0.x/",
+          "/docs/rkaas/1.0.x/": "https://github.com/rcbops/mk8s/1.0.x",
```

Comment:
do you need both here?

Reply:
we have separate lines for v13, v12 and so on above. Since these are separate branches, I figured we need both. Going forward we will probably remove 1.0.x, but for now we should keep it.